src/ensembl/src/shared/hooks/useHover.tsx

```diff
@@ -44,7 +44,7 @@ export default function useHover<T extends HTMLElement>(): UseHoverType<T> {
       element.addEventListener('mouseenter', handleMouseEnter);
       element.addEventListener('mouseleave', handleMouseLeave);
       element.addEventListener('click', handleMouseLeave);
-      element.addEventListener('touchstart', handleTouch, { passive: true });
+      element.addEventListener('touchstart', handleTouch);
```

Comment:
Why is `{passive:true}` removed here?

Reply:
sorry, good catch :-)  missed while rebasing.